### DISCUSSION TITLE
How to deal with technical replicates of scRNAseq data?

### DISCUSSION THREAD

**Discussion by SMP7UC** [satijalab/seurat#8922] [General]
Hi,

First of all, thank you so much! This platform is very helpful for me whenever I face any difficulties. Now, I have some situations for which I have not found any specific solution. 

In our study, there are six samples, 3 from control and 3 from disease. scRNAseq was done for the samples. For one of the samples from the disease group, we have two technical replicates, i.e., one sample from this group was sequenced twice (sequencing protocol was the same for both replicates). Our sequencing team informed us that their aim was to generate roughly 40K read clusters per cell. In the first sequencing, a lower # of read clusters per cell was generated, so they performed the 2nd sequencing to generate more read clusters. They suggested that two sets of data can be combined for final data analysis. Therefore, we merged those replicates and treated them as one sample, which was then integrated into other samples for downstream analysis. Since the other samples do not have any replicates, it affects analyses like cell counts, resulting in a higher standard deviation in the groups, including the sample having replicates. 

I analyzed those replicates separately and found that they are similar. Though each of the replicates has some (around 300) unique genes, their expression is very low (maximum 0.7%).

Now, we decided to select any one of those replicates with comparatively higher transcripts and lower mitochondrial genes (Which is also difficult to decide). Is this the correct way to deal with the replicates? Is there a better option in this case? Your opinion and suggestions in this regard will be greatly appreciated.

**Comment by samuel-marsh** [comment]
Hi,

Not member of dev team but hopefully can be helpful.  As this isn’t issue with Seurat package I’m going to move this to discussion section.

However I can also provide some insight.  If I’m understanding correctly you are describing the same cDNA library on two separate flowcells.  However in that case those are not technical replicates they are the same samples.

When you sequence the same library twice you cannot combine the count matrices because that fails to account for reads with the same UMI.  The runs need to be combined during read counting (e.g., `cellranger count`).  That way UMIs will be accounted for during counting.

Best,
Sam

**Reply by SMP7UC** [reply]
Hi Sam,

Thank you so much for your prompt response. 

Yes, I did not combine those during read counting. I merged those while analyzing by Seurat. Now, I understand what actually the sequencing team meant by combining those sequences.

I actually did the cell ranger analysis by 10X genomics cloud. Hopefully, it can be combined with 10x genomics cloud. 

Thank you again. It is a great learning for me.

Best,
Shrabanti
